iphone/Maps/Core/TextToSpeech/MWMTextToSpeech.mm

```diff
@@ -279,6 +282,43 @@ - (AVAudioPlayer *)audioPlayer {
   return _audioPlayer;
 }
 
+- (void)playRandomTestString {
+  if (![self isValid])
+    [self createVoice:[[self class] savedLanguage]];
+  
+  if (self.testStrings == nil)
+    return;
+  
+  [self speakOneString: self.testStrings[_testStringIndex]];
+  
+  _testStringIndex++;
+  if (_testStringIndex >= self.testStrings.count)
+    _testStringIndex = 0;
+}
+
+- (NSArray *)testStrings {
+  NSString * language = [NSString stringWithUTF8String: bcp47ToTwineLanguage([[self class] savedLanguage]).c_str()];
+  NSString * languagePath = [NSBundle.mainBundle pathForResource: language ofType: @"lproj"];
+  if (languagePath == nil) {
+    LOG(LWARNING,
+        ([@"Couldn't find translation file for " stringByAppendingString: language]));
```

Comment:
Maybe will be easier
```suggestion
        (LOG(LWARNING, ("Couldn't find translation file for ", language.UTF8String));
```

Reply:
Thanks I thought that's very complicated